diff_test.go

```diff
@@ -0,0 +1,44 @@
+package manifestival_test
+
+import (
+	"testing"
+
+	. "github.com/manifestival/manifestival"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+func TestDiff(t *testing.T) {
+	client := testClient()
+	old, _ := NewManifest("testdata/k-s-v0.11.0.yaml", UseClient(client))
```

Comment:
Good call. I want to refactor a bit first, though.

Reply:
Done. I thought it would be easier to grok by putting the current, modified, and expected output in files. Let me know if you think I missed any edge cases. Thanks!